terraform/healthcheck.tf

```diff
@@ -0,0 +1,7 @@
+module "healthcheck" {
+  source = "./healthcheck"
+
+  environment = var.environment
+
+  alerter = module.alerter.alerter_lambda
```

Comment:
Why does the healthcheck.tf need the alerter module?

Reply:
This is so the cloudwatch alarm can action the alerter lambda function using its arn.